### ISSUE TITLE
MSYS2 cmake path error

### ISSUE THREAD

**Issue by Cregrant** [facebook/redex#661]
`python3: can't open file '/c/redex-main/build-cmake/C:/redex-main/gen_packed_apilevels.py': [Errno 2] No such file or directory
make[2]: *** [CMakeFiles/generated_apilevels.dir/build.make:87: generated_apilevels.py] Error 2
make[1]: *** [CMakeFiles/Makefile2:196: CMakeFiles/generated_apilevels.dir/all] Error 2`

My working directory is /c/redex-main/build-cmake
Line 87 of build.make: `python3 C:/redex-main/gen_packed_apilevels.py -o generated_apilevels.py ...`

**Comment by agampe** [comment]
Can you list all the steps you have been doing up to that point?

*(The CI build is working just fine at this point.)*

**Comment by Cregrant** [comment]
I am using experimental Windows MSYS2 build.
```
# In a MingW64 shell:
git clone https://github.com/facebook/redex.git
cd redex
# Assumes you are in the redex directory
mkdir build-cmake
cd build-cmake
cmake -G "MSYS Makefiles" ..
make -j4                              <---- step "generated_apilevels.dir" error
```

In MSYS2 MinGW x64 shell "generated_apilevels.dir" step run
`python3 C:/redex/gen_packed_apilevels.py -o generated_apilevels.py ...`
command from _C:/redex/build-cmake_ working directory which results in 
`python3 /c/redex/build-cmake/C:/redex/gen_packed_apilevels.py -o generated_apilevels.py ...`
for some reason.

It works fine if I change the command in file to
`python3 ../gen_packed_apilevels.py -o generated_apilevels.py ...`

**Comment by agampe** [comment]
The paths look wonky. I was told they look like they were generated with Visual Studio creating the Makefiles. I don't know why that happens for you.

Did you run *all* steps in the MinGW shell?

**Comment by Cregrant** [comment]
Yes. I have no idea why absolute paths are converted to relative but it might be an issue in a shell [like this](https://github.com/golang/go/issues/46077).
